.eslintrc.json

```diff
@@ -14,7 +14,8 @@
   "globals": {
     "flowchart": true, // Blueprints support lib
     "__": true, // Zanata parsing cue
-    "N_": true // Zanata parsing cue
+    "N_": true, // Zanata parsing cue
+    "$window": true
```

Comment:
I think you can avoid declaring `$window` as a global if you dependency inject it at `client/app/config/authorization.config.js:60`.